templates/pages/category.html

```diff
@@ -24,11 +24,13 @@ <h1 class="page-heading">{{category.name}}</h1>
 {{{category.description}}}
 {{{snippet 'categories'}}}
 <div class="page">
-    {{#or category.subcategories category.faceted_search_enabled category.shop_by_price}}
-        <aside class="page-sidebar" id="faceted-search-container">
-            {{> components/category/sidebar}}
-        </aside>
-    {{/or}}
+    {{#if theme_settings.shop_by_price_visibility}}
+        {{#or category.subcategories category.faceted_search_enabled category.shop_by_price}}
```

Comment:
pretty sure this is not the correct solution because shop by price could be off while these other features are on correct?